src/molecules/BundleDealBox/BundleDealBox.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import Button from '../../atoms/Button';
+import Heading from '../../atoms/Heading';
+
+type FlashText = {
+  text: string;
+  textColor: string;
+  color: string;
+};
+
+type BundleDealBoxProps = {
+  flashText?: FlashText;
+  image: {
+    url: string;
+  };
+  productName: string;
+  description?: string;
+  discountPrice: string;
+  salesPrice: string;
+  disclaimer?: string;
+  buttonText?: string;
+  url: string;
+};
+
+const BundleDealBox: React.FC<BundleDealBoxProps> = ({
+  flashText,
+  image,
+  productName,
+  description,
+  discountPrice,
+  salesPrice,
+  disclaimer,
+  buttonText,
+  url,
+}) => {
+  return (
+    <div className="bundleDealBox">
+      <div className="bundleDealBox__flashText-container">
+        {flashText && (
+          <div
+            className="bundleDealBox__flashText"
+            style={{ color: flashText.textColor, backgroundColor: flashText.color }}
+          >
+            {flashText.text}
+          </div>
+        )}
+      </div>
+      <div className="bundleDealBox__image-container">
+        <img loading="lazy" className="bundleDealBox__product-image" src={`${image.url}?w=180`} alt="" />
+      </div>
+      <div className="bundleDealBox__upper-container">
+        <Heading tag="h2" size="s" text={productName} />
+        {description && (
+          <div className="bundleDealBox__description paragraph" dangerouslySetInnerHTML={{ __html: description }} />
+        )}
+        <div className="bundleDealBox__price-container">
+          <span className="bundleDealBox__price-container--strikethrough">{discountPrice}</span>
+          <span className="bundleDealBox__price-container--salesPrice">{salesPrice}</span>
+        </div>
+        <div>{disclaimer}</div>
+        <Button className="bundleDealBox__link" component="link" href={url} text={buttonText} />
```

Comment:
```suggestion
       {buttonText && (
          <Button className="bundleDealBox__link" component="link" href={url} text={buttonText} />
       )}
```
Since `buttonText` is optional